components/Clock.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="clock">
+    <div class="date-time">{{ currentDate }} | {{ timer }}{{ clock() }}</div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Clock',
+  data() {
+    return {
+      currentDate: new Date().toDateString(),
+      timer: '',
+    }
+  },
+  methods: {
+    myTimer() {
+      return new Date().toLocaleTimeString()
+    },
+    clock() {
+      let h = new Date().getHours()
+      let m = new Date().getMinutes()
+      let session = 'am'
+      if (h > 12) {
+        h = h - 12
+        session = 'pm'
+      }
+      h = h < 10 ? '0' + h : h
+      m = m < 10 ? '0' + m : m
+      const formatedText = h + ':' + m + session + ' ' + 'IST'
+      this.timer = formatedText
+      setTimeout(this.clock, 1000)
+    },
+  },
+}
+</script>
+
+<style scoped>
+@import url('https://fonts.googleapis.com/css2?family=Saira+Semi+Condensed:wght@600&display=swap');
```

Comment:
Why is this font scoped? Shouldn't it be available to other components in the application?

Reply:
done